README.md

```diff
@@ -84,6 +84,8 @@ After you have selected the appropriate options, click the "Deploy Now" button t
 ## Step 2: Install the SNI Proxy
 Once your server has been created, log in to the server using SSH or console. The root password is available under the "Overview" tab in instances list.
 
+Ensure the firewall is disabled or ports 80, 443 and 53 are open. For example on Vultr's Ubuntu instance run `sudo ufw disable`.
```

Comment:
as a security guy I'm not really comfortable suggesting a blanket "disable firewall" comments. I suggest we change it to:

Ensure your firewall (firewalld/ufw/iptables) is allowing connectivity to the TCP and UDP ports as needed. by default, 80/TCP, 443/TCP and 53/UDP need to be accessible from the internet. for `ufw`, use the following commands:
``` bash
sudo ufw allow 80/tcp
sudo ufw allow 443/tcp
sudo ufw allow 53/udp
sudo ufw reload
```